python/nx-cugraph/README.md

```diff
@@ -45,18 +45,18 @@ Notes:
 NetworkX will use nx-cugraph as the graph analytics backend if any of the
 following are used:
 
-### `NETWORKX_AUTOMATIC_BACKENDS` environment variable.
-The `NETWORKX_AUTOMATIC_BACKENDS` environment variable can be used to have NetworkX automatically dispatch to specified backends an API is called that the backend supports.
-Set `NETWORKX_AUTOMATIC_BACKENDS=cugraph` to use nx-cugraph to GPU accelerate supported APIs with no code changes.
+### `NX_CUGRAPH_AUTOCONFIG` environment variable.
+The `NX_CUGRAPH_AUTOCONFIG` environment variable can be used to have NetworkX automatically dispatch to specified backends if an API is called that the backend supports.
+Set `NX_CUGRAPH_AUTOCONFIG=True` to use nx-cugraph to GPU accelerate supported APIs with zero code change.
```

Comment:
I think we should update the language to say this specifically automatically dispatches to nx-cugraph. Previously, this section discussed `NETWORKX_AUTOMATIC_BACKENDS` which is more generic.

Also, do you think we should share _how_ this works? That is, it works by setting or updating environment variables that control the config: https://networkx.org/documentation/stable/reference/backends.html . It's also fine if we want to keep the README minimal, but someday we may want a reference page that explains how.

Reply:
I just updated the README based on your feedback.

I updated the URL at the top to -> https://networkx.org/documentation/stable/backends.html

And for the _environment variable_ section, I updated the description on `NX_CUGRAPH_AUTOCONFIG` and linked to the Backends page so users can read more on how backends work.

Reply:
See new lines 49-51